src/main/java/co/omise/resources/RefundResource.java

```diff
@@ -22,23 +22,6 @@ public String chargeId() {
         return chargeId;
     }
 
-    public ScopedList<Refund> list() throws IOException, OmiseException {
-        return list(new ScopedList.Options());
-    }
-
-    public ScopedList<Refund> list(ScopedList.Options options) throws IOException, OmiseException {
-        return httpGet(urlFor("")).params(options).returns(new TypeReference<ScopedList<Refund>>() {
-        });
-    }
-
-    public Refund get(String refundId) throws IOException, OmiseException {
-        return httpGet(urlFor(refundId)).returns(Refund.class);
-    }
-
-    public Refund create(Refund.Create params) throws IOException, OmiseException {
-        return httpPost(urlFor("")).params(params).returns(Refund.class);
-    }
-
```

Comment:
Why not delete the whole class?

Reply:
@Namiii `ChargeSpecificResource` still using `RefundResource`. Not sure it safe to delete or not? 🤔

Reply:
No you're right, we'll remove it once that resource is also removed 👍